services/src/main/scala/cromwell/services/metadata/impl/MetadataSummaryRefreshActor.scala

```diff
@@ -29,13 +29,17 @@ object MetadataSummaryRefreshActor {
   sealed trait SummaryRefreshState
   case object WaitingForRequest extends SummaryRefreshState
   case object SummarizingMetadata extends SummaryRefreshState
-  case object MetadataSummaryComplete extends SummaryRefreshState
 
-  case object SummaryRefreshData
+  sealed trait SummaryRefreshData
+  case object EmptySummaryRefreshData extends SummaryRefreshData
+  final case class KnownNextSummaryStatusPointerCeiling(ceiling: Long) extends SummaryRefreshData
```

Comment:
standardize terminology plz: `KnownNextSummaryStatusPointerCeiling` vs `nextSummaryPointerUpdateCeiling`

Reply:
This is a `StateData` type for an actor, so this represents that the actor has a "known" `nextSummaryPointerUpdateCeiling`. I'll see if I can come up with a better naming scheme